.github/workflows/maven.yml

```diff
@@ -53,8 +53,13 @@ jobs:
         id: sdkman
 
       - name: 'Build project'
+        shell: bash
         run: |
           source "$HOME/.sdkman/bin/sdkman-init.sh"
+          if [ -f ${{ format('src/main/java-22/dev/morling/onebrc/CalculateAverage_{0}.java', github.event.pull_request.user.login || '') }} ]; then
+            sdk install java 22.ea.32-open || true
+            sdk use java 22.ea.32-open
+          fi
```

Comment:
This should not be needed, the `evaluate.sh` script takes care of installing all required JDKs.

Reply:
`test_ci.sh` installs the JDKs but it runs after maven so without that line, my class under `java-22/` wouldn't be compiled.

Here's the original run without the change
https://github.com/gunnarmorling/1brc/actions/runs/7724135717/job/21055662835